.github/workflows/tests.yaml

```diff
@@ -0,0 +1,55 @@
+name: Tests
+
+on:
+    push:
+    pull_request:
+        branches: [ master ]
+    workflow_dispatch:
+
+jobs:
+    phpunit:
+        runs-on: ubuntu-latest
+        strategy:
+            fail-fast: false
+            matrix:
+                php: [ '8.1', '8.2' ]
+                symfony: [ '5.4', '6.2' ]
+                dependencies: [ 'highest', 'lowest' ]
+        env:
+            SYMFONY_REQUIRE: ${{ matrix.symfony }}.*
+        name: PHP ${{ matrix.php }} & Symfony ${{ matrix.symfony }}${{ matrix.dependencies == 'lowest' && ' (lowest)' || '' }} Test
+        steps:
+            -   name: Checkout
+                uses: actions/checkout@v3
+
+            -   name: Setup PHP
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: ${{ matrix.php }}
+                    ini-values: zend.exception_ignore_args=false
+                    tools: flex
+
+            -   name: Validate "composer.json" schema
+                run: composer validate --ansi
+
+            -   name: Install dependencies
+                uses: ramsey/composer-install@v2
+                with:
+                    composer-options: '--prefer-dist'
+                    dependency-versions: ${{ matrix.dependencies }}
+
+            -   name: Install PHPUnit dependencies
+                run: vendor/bin/simple-phpunit install
+
+            -   name: Run tests
+                run: vendor/bin/simple-phpunit --colors=always --coverage-clover=clover.xml --printer=mheap\\GithubActionsReporter\\Printer --verbose
```

Comment:
@HypeMC `composer test -- ...` can be used here instead.

Reply:
`composer test` is a nice shortcut when you're running tests locally, but since we use additional flags on the CI, there's no real benefit in using it here as well IMO.